apps/src/maze/maze.js

```diff
@@ -1601,8 +1601,8 @@ function setPegmanTransparent() {
  * @param {integer} timeAlloted How much time we have for our animations
  */
 function scheduleDance(victoryDance, timeAlloted) {
-  if (mazeUtils.isScratSkin()) {
-    scrat.scheduleDance(victoryDance, timeAlloted);
+  if (mazeUtils.isScratSkin(skin.id)) {
+    scrat.scheduleDance(victoryDance, timeAlloted, skin);
```

Comment:
Np, it was my bug. I did a refactor at one point, apparently without properly testing the code. Jshint would have caught it if we had undef enabled ;)